pocs/camera/camera.py

```diff
@@ -17,6 +19,9 @@ def __init__(self,
                  model='simulator',
                  port=None,
                  primary=False,
+                 set_point=None,
```

Comment:
OK, fair point, I'll take that out.

The SBIG `Camera` class has `CCD_set_point` as a property and implements getter and setter methods. Would it make sense to include stub versions of these in the `AbstractCamera` base class, to encourage a consistent interface for cameras with cooling controls?